pkgs/development/python-modules/django-upgrade/default.nix

```
@@ -0,0 +1,42 @@
+{
+  buildPythonPackage,
+  fetchFromGitHub,
+  lib,
+  pytestCheckHook,
+  setuptools,
+  tokenize-rt,
+  versionCheckHook,
+}:
+
+buildPythonPackage rec {
```

```suggestion
buildPythonApplication rec {
```
I think this package is an application but not a library.